csxtools/settings.py

```diff
@@ -1,3 +1,4 @@
 detectors = {}
-detectors['fccd'] = 'fccd_image'
-diff_angles = ['delta', 'theta', 'gamma', None, None, None]
+detectors["fccd"] = "fccd_image"
+detectors["axis1"] = "axis1_image"
```

Comment:
this looks like it will be a problem, because we actually have multiple axis detectors.  We may need to make a separate issue to deal with this.  @nisarnk please make an issue with this line module so we can investigate it and develop the most generalized solution.

Reply:
these are correct tags in use so far, but there will be more to come and perhaps it will change.  This is why I am suggesting to use `header.start["detectors"]` if `tag = None` when calling `get_axis_images()` .  I need to better understand how settings.py is being used.  It looks more like for reciprocal space calculations and not anything to do with  normalizing area detector data.  It is unlikely that we will have `fccd` data and `axis` data in the same scan so exceptions will occur.

Let me take a 20 minute to look at the main branch and understand what `settings.py` does

Reply:
OK, now I see what is happening.  I guess calling header.start["detectors"] and making inferences was going to be slower.  So here, we either do this for axis OR we put all the possibilities in a list and integrate ignoring `KeyErrror` exceptions until we get to the end of the list.  We should decide which is better for axis.

Reply:
Resolve as per https://github.com/NSLS-II-CSX/csxtools/issues/100 
Code is conservative to prevent unwanted KeyErrors